convert2rhel/utils.py

```diff
@@ -60,9 +60,9 @@ def get_executable_name():
 
 def require_root():
     if os.geteuid() != 0:
-        loggerinst = logging.getLogger(__name__)
-        loggerinst.critical("The tool needs to be run under the root user.")
-    return
+        print "The tool needs to be run under the root user."
+        print("\nNo changes were made to the system.")
+        sys.exit(1)
```

Comment:
Should be reverted as mentioned earlier

Reply:
Actually that's was intentional.. if you use loggerinst, will raise a exception, because they will try to open the log file without permission, that's the reason to move to print.

Reply:
Ack, then one simple fix for print:
```suggestion
        print("The tool needs to be run under the root user.")
        print("\nNo changes were made to the system.")
        sys.exit(1)
```

Reply:
Still not work, when you initialize the logger, they try to open the log file... And as these message does not need to be logged (since it's only a message before to run the tool), the print is a good idea.. 
Fixing forgot `(`..